_posts/2023-10-22-specify-file-audit-log-encoding.adoc

```diff
@@ -0,0 +1,89 @@
+---
+layout: post
+title: 'Change the default encoding of the audit log file in WildFly 29'
+date: 2023-10-22
+tags: audit-logging encoding
+synopsis:  An overview of how to change the default encoding of the audit log file.
+author: ivassile
+---
+
+The default encoding used for the audit log file is UTF-8. You can change the default encoding by specifying `encoding` attribute in `file-audit-log`, `periodic-rotating-file-audit-log` or `size-rotating-file-audit-log` elements in the Elytron subsystem. 
+Possible values are: `UTF-8`, `UTF-16BE`, `UTF-16LE`, `UTF-16`, `US-ASCII` or `ISO-8859-1`.
+
+[source,xml]
+----
+<file-audit-log name="local-file" path="audit.log" relative-to="jboss.server.log.dir" synchronized="false" autoflush="true" format="JSON" encoding="UTF-16"/>
+...
+<periodic-rotating-file-audit-log name="periodic-rotating" path="audit.log" relative-to="jboss.server.log.dir" format="JSON" encoding="US-ASCII" suffix="y-M-d"/>
+...
+<size-rotating-file-audit-log name="size-rotating" path="audit.log" relative-to="jboss.server.log.dir" format="JSON" encoding="ISO-8859-1" max-backup-index="5" rotate-on-boot="true" rotate-size="5" suffix="y-M-d"/>
+----
+
+== Example
+
+This example will show how to deploy a simple web application, update the security domain configuration to enable audit logging and inspect the resulting file.
+We will use the simple-webapp example which can be found https://github.com/wildfly-security-incubator/elytron-examples/tree/main[here].
+
+*Clone the `elytron-examples` repo locally:*
+
+```
+git clone https://github.com/wildfly-security-incubator/elytron-examples
+
+cd elytron-examples
+
+```
+
+*Server configuration*
```

Comment:
Instead of *Server configuration*, I think === Server configuration should be used to get the formatting right (if you rename your branch to develop and push your changes, you should also be able to preview the post from your own fork).